benchmarks/src/main/java/org/apache/druid/benchmark/FlattenJSONBenchmarkUtil.java

```diff
@@ -54,7 +54,7 @@ public FlattenJSONBenchmarkUtil()
   public Parser getFlatParser()
   {
     JSONParseSpec spec = new JSONParseSpec(
-        new TimestampSpec("ts", "iso", null),
+        new TimestampSpec("ts", "iso", null, null),
```

Comment:
It might have been worth considering adding an additional constructor to `TimestampSpec` with `@VisibleForTesting` annotation so that this PR could have a bit lighter of a footprint and not have to modify all of these tests since they are all just passing in `null` anyway.

Reply:
I think I should learn how to use the annotation @VisibleForTesting ~

Reply:
Will it be a bit confusing for developers?
```
  @VisibleForTesting
  public TimestampSpec(
      @JsonProperty("column") String timestampColumn,
      @JsonProperty("format") String format,
      @JsonProperty("missingValue") DateTime missingValue
  )
  {
    this.timestampColumn = (timestampColumn == null) ? DEFAULT_COLUMN : timestampColumn;
    this.timestampFormat = format == null ? DEFAULT_FORMAT : format;
    this.timeZone = DateTimes.UTC_TIMEZONE;
    this.timestampConverter = TimestampParser.createObjectTimestampParser(timestampFormat, this.timeZone);
    this.missingValue = missingValue == null
                        ? DEFAULT_MISSING_VALUE
                        : missingValue;
  }
```